be/src/exprs/string_functions.cpp

```diff
@@ -3030,61 +3027,6 @@ int StringFunctions::index_of(const char* source, int source_count, const char*
     return -1;
 }
 
-struct StringFunctionsState {
```

Comment:
Why move it to header file? Does not that will increase compilation time?

Reply:
we need this struct for ut and benchmark, so move it to header and included by ut and bench.